src/components/common/vatsim/CommonEventCard.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+    <div>
+        <div
+            class="event-card"
+            :class="{ 'active-event': active }"
```

Comment:
Let's rename this to original class modifier

```suggestion
            :class="{ 'event-card--active': active }"
```